src/main/java/com/itspace/job/endpoint/UserEndPoint.java

```diff
@@ -0,0 +1,33 @@
+package com.itspace.job.endpoint;
+
+import com.itspace.job.dto.RegisterUserDto;
+import com.itspace.job.model.User;
+import com.itspace.job.service.UserService;
+import org.modelmapper.ModelMapper;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class UserEndPoint {
+
+    private UserService userService;
+    private ModelMapper mapper;
+
+    public UserEndPoint(UserService userService, ModelMapper mapper) {
+        this.userService = userService;
+        this.mapper = mapper;
+    }
+
+    @PostMapping("/register")
+    public ResponseEntity<RegisterUserDto> register(@RequestBody User user) {
```

Comment:
please create some dto for @RequestBody
don't use entity object
create some dto and add required fields in the dto class.